src/main/java/seedu/address/logic/Messages.java

```diff
@@ -18,8 +18,8 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid!";
     public static final String MESSAGE_INVALID_COMPANY_DISPLAYED_INDEX = "The company index provided is invalid!";
-    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
-    public static final String MESSAGE_COMPANIES_LISTED_OVERVIEW = "%1$d companies listed!";
+    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d person(s) listed!";
+    public static final String MESSAGE_COMPANIES_LISTED_OVERVIEW = "%1$d company(ies) listed!";
```

Comment:
Will this change the output of the UI?

Reply:
This will change the output. Not sure if this is allowed

Reply:
Following what PRof has said, I don't think we are allowed to change this

Reply:
I reverted the modification